components/style/properties/properties.mako.rs

```diff
@@ -65,6 +65,48 @@ pub trait MaybeBoxed<Out> {
     fn maybe_boxed(self) -> Out;
 }
 
+
+/// Addresses bug 16518. The cached_system_font
```

Comment:
No need to mention which bug this addresses.

Just say the last part; "This is where we store extra font data ..."